src/components/Day.vue

```diff
@@ -1,22 +1,61 @@
 <template>
   <div class="day">
     <header class="day__header">{{day | moment("dddd, MMM Do YYYY")}}</header>
-    <div class="day__content">
-      <slot name="timeline" />
+    <div class="day__content" >
+      <timeline slot="timeline"  :showLabels="showLabels" v-on:timelineclick="createEntry"/>
```

Comment:
pass entries for days to the timeline so it can display them 'on top of grid'?
Timeline could also calculate spread from start to finish of the same issue key?